src/mbgl/actor/scheduler.cpp

```diff
@@ -41,4 +41,30 @@ std::shared_ptr<Scheduler> Scheduler::GetBackground() {
     return scheduler;
 }
 
+// static
+std::shared_ptr<Scheduler> Scheduler::GetSequenced() {
+    const std::size_t kSchedulersCount = 10;
```

Comment:
Why 10?

Reply:
it's just an upper bound for the possible amount of threads, can be any other number.